clients/ui/frontend/src/concepts/modelCatalog/const.ts

```diff
@@ -326,17 +343,26 @@ export enum AllLanguageCode {
 
 /**
  * Display names for filter categories.
- * TODO: When performance filters are ready, switch this to be a Record<ModelCatalogFilterKey, string>
- * to include all ModelCatalogFilterKeys (ModelCatalogStringFilterKey | ModelCatalogNumberFilterKey).
- * This will allow separate filter category names for "Max latency (TTFT Mean)" and "Max latency (TTFT P99)" etc.
+ * Includes all ModelCatalogFilterKeys (ModelCatalogStringFilterKey | ModelCatalogNumberFilterKey | LatencyMetricFieldName).
  */
-export const MODEL_CATALOG_FILTER_CATEGORY_NAMES: Record<ModelCatalogStringFilterKey, string> = {
+export const MODEL_CATALOG_FILTER_CATEGORY_NAMES: Record<
+  ModelCatalogStringFilterKey | ModelCatalogNumberFilterKey | LatencyMetricFieldName,
```

Comment:
Let's have this use the type `ModelCatalogFilterKey` which is the same, and that will force us to update this if we introduce new filter key types later.